shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -792,15 +795,24 @@ public long getRequestId() {
         }
     }
 
-    @Data
+    @RequiredArgsConstructor(access = AccessLevel.PRIVATE)
+    @Getter
+    @ToString
+    @EqualsAndHashCode(exclude = {"mustRelease", "released"})
```

Comment:
NotThreadSafe unless you want to make released an atomic boolean

Reply:
Done.